app/controllers/profiles_controller.rb

```diff
@@ -1,7 +1,6 @@
 class ProfilesController < ApplicationController
-  skip_before_action :authenticate_user!, except: %i[intro new create]
```

Comment:
Does removing this mean that someone can't access some of the actions (like the `index` and `solutions` actions) without being logged in?